lib/httpDigest.js

```diff
@@ -19,13 +22,64 @@ async function create({data, algorithm = 'sha256', useMultihash = true}) {
   if(typeof data !== 'string') {
     data = JSON.stringify(data);
   }
+  algorithm = algorithm.toLowerCase();
+  const digest = await _getDigest({data, algorithm});
+  if(useMultihash) {
+    return _createMultiHash({digest});
+  }
+  return `SHA-256=${base64Encode(digest)}`;
+}
+
+/**
+ * Verifies digest of data created by create().
+ *
+ * @param {object} options - The options to use.
+ * @param {string|object} options.data - The data to be verified.
+ * @param {string} options.header - The digest header string to
+ *   be verified against.
+ *
+ * @returns {object} - Returns an object with verified true or false.
+ */
+async function verify({data, header}) {
```

Comment:
Similar fix needed above on `create`:

```suggestion
async function verify({data, header} = {}) {
```